analysis/report_templates/normalized_score.html

```diff
@@ -0,0 +1,238 @@
+<!--
```

Comment:
Can you move this report addition change to a new CL, new branch (git pull, so that old patches dont show in PR). This CL should be a clean version of just removing AFL variants.

Reply:
Also, i dont think this file needs to be copied, i think there should be a way to just change method name in the report. Code duplication is bad. probably just remove this change and @lszekeres can help with adding this normalized score method.

Reply:
currently it needs to be copied. I agree an option for the command line would be better. Removed the changes from this PR